docs/design/opm-tooling.md

```diff
@@ -93,6 +93,41 @@ Like `opm registry rm`, this command will remove all versions an entire operator
 
 This will result in the tagged container image `quay.io/operator-framework/monitoring-index:1.0.2` with a registry that no longer contains the `prometheus` operator at all.
 
+#### export
+
+`opm index export` will export a package from an index into a directory. The format of this directory will match the appregistry manifest format: containing all versions of the package in the index along with a `package.yaml` file. This command takes an `--index` flag that points to an index image, a `--package` flag that states a package name, an optional `--download-folder` as the export location (default is `./downloaded`), and just as the other index commands it takes a `--container-tool` flag. Ex:
```

Comment:
You mean "an index image". We have been unclear about "index" and "index image". I feel in this case you meant "index image". Is that right?